dracut/30ignition/coreos-teardown-initramfs.sh

```diff
@@ -162,6 +179,10 @@ main() {
     # clean it up so that no information from outside of the
     # real root is passed on to NetworkManager in the real root
     rm -rf /run/NetworkManager/
+
+    # If automated multipath configuration has been enabled, ensure
+    # that its propagated to the real rootfs.
+    persist_automatic_multipath
```

Comment:
nit: the other functions are called propagate_initramfs*. suggest: propagate_initramfs_multipath